homeassistant/components/abode/__init__.py

```diff
@@ -96,7 +92,7 @@
 class AbodeSystem:
     """Abode System class."""
 
-    def __init__(self, username, password, cache, name, polling, exclude, lights):
+    def __init__(self, username, password, cache, name, polling):
         """Initialize the system."""
         import abodepy
```

Comment:
Please create the Abode instance before passing it to the AbodeSystem class. We want to avoid side effects in init method.